crates/sui-indexer/tests/read_api_tests.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use jsonrpsee::core::RpcResult;
+use simulacrum::Simulacrum;
+use std::sync::Arc;
+use sui_indexer::apis::read_api::ReadApi;
+use sui_indexer::indexer_reader::IndexerReader;
+use sui_indexer::test_utils::{set_up, wait_for_checkpoint};
+use sui_json_rpc_api::ReadApiServer;
+use tempfile::tempdir;
+
+#[tokio::test]
+async fn test_read_api() -> RpcResult<()> {
```

Comment:
Can we rename this test to be more descriptive that its testing the checkpoint functionality of the ReadApi and not any other apis?